vnda/hooks/context.ts

```diff
@@ -1,15 +1,18 @@
 import { IS_BROWSER } from "$fresh/runtime.ts";
 import { signal } from "@preact/signals";
 import { invoke } from "../runtime.ts";
+import type { Person } from "../../commerce/types.ts";
 import type { Cart } from "../loaders/cart.ts";
 
 export interface Context {
+  user: Person | null;
   cart: Cart;
 }
 
 const loading = signal<boolean>(true);
 const context = {
-  cart: signal<Cart | null>(null),
+  user: IS_BROWSER && signal<Person | null>(null) || { value: null },
+  cart: IS_BROWSER && signal<Cart | null>(null) || { value: null },
```

Comment:
Why this verification its necessary?

Reply:
The `IS_BROWSER` check ensures compatibility with both browser and server environments.

- **Client-side**: When `IS_BROWSER` is `true`, reactive signals like `signal<OrderForm | null>(null)` are used for dynamic UI updates.

- **Server-side**: When `IS_BROWSER` is `false`, simple objects `{ value: null }` are used instead, as reactivity isn't needed or available.

This allows the code to work efficiently in both contexts. These are good practices that Deco engineers brought. The VTEX context also makes use of this check